recipes/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "python-moa" %}
```

Comment:
This should be in it's own folder called `python-moa`. Also can you restore the example recipe.

Reply:
Fixed. Is this now the correct structure? I'm new to submitting a package to conda-forge and not familiar with the structure.